configure.py

```diff
@@ -341,6 +341,8 @@ def search_system_path(file_name: str) -> Optional[str]:  # type: ignore # Missi
               '/wd4355',
               # Disable warnings about ignored typedef in DbgHelp.h
               '/wd4091',
+              # Disable warning about signed/unsigned mismatch and _unlink in gtest
```

Comment:
Is it possible to fix the underlying source issues instead of disabling warnings (which has the potential of hiding bugs)?